packages/insomnia-testing/src/run/run.ts

```diff
@@ -23,15 +23,15 @@ const runInternal = async <T>(
   // eslint-disable-next-line @typescript-eslint/no-explicit-any -- type not available, and postponing anyway until the impending move to all jest (and no mocha)
   extractResult: (runner: { [key: string]: any }) => T,
 ): Promise<T> => new Promise((resolve, reject) => {
-  const { bail, keepFile, testFilter } = options;
+  const { bail, keepFile, testFilter, timeout } = options;
 
   // Add global `insomnia` helper.
   // This is the only way to add new globals to the Mocha environment as far as I can tell
   global.insomnia = new Insomnia(options);
   global.chai = chai;
 
   const mocha: Mocha = new Mocha({
-    timeout: 5000,
+    timeout: timeout || 5000,
```

Comment:
nit: Consider using `??`, otherwise a `timeout` value of 0 would be coerced to 5000. I'm not sure it's ever particularly useful, but a value of 0 disables timeouts.

Reply:
Oh this is a great catch, TY. Not tested yet so still draft 😆 

```suggestion
    timeout: timeout ?? 5000,
```